ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,15 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+namespace DigitalRootFinder {
+   class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter the number:");
+         int n = int.Parse (Console.ReadLine ());
+         if (n != 0)

```

Comment:
What if input is 0?

Reply:
will give with a if else statement and update